sockeye/arguments.py

```diff
@@ -317,6 +317,20 @@ def add_model_parameters(params):
     model_params.add_argument('--transformer-no-positional-encodings',
                               action='store_true',
                               help='Do not use positional encodings.')
+    model_params.add_argument('--transformer-preprocess',
+                              type=multiple_values(num_values=2, greater_or_equal=None, data_type=str),
+                              default=('', ''),
+                              help='Transformer preprocess sequence for encoder and decoder.'
+                                   'For example ":drn" would refer to no preprocessing on the encoder and (d)ropout, '
```

Comment:
The example is a bit confusing because of the empty part for the encoder. You could have an example like "d:drn" and some note like "leave a part empty for no preprocessing". Also, it would be good to list all the possible options. If it's only 'drn', then state this explicitely.

Reply:
This gets a bit long in my opinion if I state all combinations of three letters (also it could be longer than three). I will update to make it a bit clearer. But this is an expert setting. One has to go into the model architecture details to understand what you want to do :)

Reply:
By options I meant 'processing' options. Something like "operations can be chosen from the set {d,r,n}".